Description: Don't run ldconfig after calling the install target
 and save a bit of time.
Author: Alessio Treglia <alessio@debian.org>
Forwarded: not-needed
---
 wscript |    1 -
 1 file changed, 1 deletion(-)

Index: serd/wscript
===================================================================
--- serd.orig/wscript
+++ serd/wscript
@@ -266,9 +266,6 @@ def build(bld):
     # Man page
     bld.install_files('${MANDIR}/man1', 'doc/serdi.1')
 
-    bld.add_post_fun(autowaf.run_ldconfig)
-
-
 class LintContext(Build.BuildContext):
     fun = cmd = 'lint'
 
